include/mbedtls/sha512.h

```diff
@@ -212,18 +229,27 @@ extern "C" {
 #endif
 
 /**
- * \brief          Output = SHA-512( input buffer )
+ * \brief          This function calculates the SHA-512 or SHA-384 
+ *                 checksum of a buffer.
+ *
+ *                 The function allocates the context, performs the 
+ *                 calculation, and frees the context.
+ *
+ *                 The SHA-512 result is calculated as 
+ *                 output = SHA-512(input buffer).
  *
- * \param input    buffer holding the data
- * \param ilen     length of the input data
- * \param output   SHA-384/512 checksum result
- * \param is384    0 = use SHA512, 1 = use SHA384
+ * \param input    The buffer holding the input data.
+ * \param ilen     The length of the input data.
+ * \param output   The SHA-384 or SHA-512 checksum result.
+ * \param is384    Determines which function to use.
+ *                 <ul><li>0: Use SHA-512.</li>
+ *                 <li>1: Use SHA-384.</li></ul>
  *
- * \return         0 if successful
+ * \return         \c 0 on success.
  */
-int mbedtls_sha512_ret( const unsigned char *input,
+int mbedtls_sha512_ret( const unsigned char *input, 
                         size_t ilen,
-                        unsigned char output[64],
+                        unsigned char output[64], 
```

Comment:
Blocker: Please remove the trailing space.

Reply:
Done